internal/proxy/task_query.go

```diff
@@ -169,6 +173,74 @@ func parseQueryParams(queryParamsPair []*commonpb.KeyValuePair) (*queryParams, e
 	}, nil
 }
 
+func matchCountRule(outputs []string) bool {
+	return len(outputs) == 1 && outputs[0] == "count(*)"
```

Comment:
what about  "    count(*)     "?
Do we think it is legal after trim?

Reply:
OK, I agree.